tools/toolchain/dbuild

```diff
@@ -155,7 +155,9 @@ if [ -z "$is_podman" ]; then
        )
 else
     docker_common_args+=(
-        --userns keep-id)
+        --userns keep-id
+        --systemd=always
```

Comment:
>               • Podman sets the default stop signal to SIGRTMIN+3.

Please verify that ctrl-C still works during build and test.py execution. If not we may need to override --stop-signal.

Reply:
It's not working on master branch either. Tried to add `--stop-signal=SIGTERM` and `--stop-signal=SIGKILL`, but still no luck.

Reply:
It's working for me with podman, but only interactive mode (invoking dbuild without parameters).

Reply:
It works when I'm already in interactive mode, when I start the build or test.py and then cancel them.